protocol/avdl/chat1/common.avdl

```diff
@@ -15,7 +15,8 @@ protocol common {
     ATTACHMENT_2,
     EDIT_3,
     DELETE_4,
-    METADATA_5
+    METADATA_5,
+    TLFNAME_6
```

Comment:
Should this be `HEADLINE_6`? We already have TlfName in `MessageClientHeader` right?